paasta_tools/docker_wrapper.py

```diff
@@ -272,6 +276,44 @@ def append_cpuset_args(argv, env_args):
     return argv
 
 
+def add_firewall(argv, service, instance):
+    output = ''
+    sensu_status = pysensu_yelp.Status.OK
+
+    try:
+        mac_address, lockfile = reserve_unique_mac_address(LOCK_DIRECTORY)
+    except Exception as e:
+        output = 'Unable to add mac address: {}'.format(e)
+        sensu_status = pysensu_yelp.Status.CRITICAL
+    else:
+        argv = add_argument(argv, '--mac-address={}'.format(mac_address))
+        try:
+
+            with firewall_flock():
+                prepare_new_container(
+                    DEFAULT_SOA_DIR,
+                    DEFAULT_SYNAPSE_SERVICE_DIR,
+                    service,
+                    instance,
+                    mac_address)
+        except Exception as e:
+            output = 'Unable to add firewall rules: {}'.format(e)
+            sensu_status = pysensu_yelp.Status.CRITICAL
+
+    if output:
+        print(output, file=sys.stderr)
+
+    send_event(
```

Comment:
I appreciate the thought to include sensu, but I'm really hesitant to give a shipit on adding this to the wrapper.

This thing has to be bullet proof. 

Even if it sends alerts like this, are they *really* actionable to the end user?

I would rather this thing just log to stderr so it will show up in the logs and output of mesos.